install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -246,6 +246,8 @@ type UsageConfig struct {
 	Enabled                          bool               `json:"enabled"`
 	Schedule                         string             `json:"schedule"`
 	BillInstancesAfter               *time.Time         `json:"billInstancesAfter"`
+	DefaultSpendingLimitForUsers     int32              `json:"defaultSpendingLimitForUsers"`
+	DefaultSpendingLimitForTeams     int32              `json:"defaultSpendingLimitForTeams"`
```

Comment:
```suggestion
	DefaultSpendingLimitForUsers     int              `json:"defaultSpendingLimitForUsers"`
	DefaultSpendingLimitForTeams     int              `json:"defaultSpendingLimitForTeams"`
```

Reply:
We use int32 everywhere for spending limit. If we want to chaneg this I'd prefer to do it in a separate PR